### PR TITLE
make URI path for getting all incentivized packets for channel consistent

### PR DESCRIPTION
**Pull request by crodriguezvega**
## Description

Just a nit to make the URI path for the query to get all incentivized packets for a specific channel consistent with the rest of queries.

Not sure if I need to add changelog for this. Please advice!

closes: #XXXX

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: proto/ibc/applications/fee/v1/query.proto [hunk 1]
```diff
@@ -29,7 +29,7 @@ service Query {
   // Gets all incentivized packets for a specific channel
   rpc IncentivizedPacketsForChannel(QueryIncentivizedPacketsForChannelRequest)
       returns (QueryIncentivizedPacketsForChannelResponse) {
-    option (google.api.http).get = "/ibc/apps/fee/v1/incentivized_packets/{port_id}/{channel_id}";
+    option (google.api.http).get = "/ibc/apps/fee/v1/incentivized_packets/port/{port_id}/channel/{channel_id}";
```
Comment (damiannolan) [inline proto/ibc/applications/fee/v1/query.proto:32]: Nice catch!

**Review by damiannolan — APPROVED** [review]
I think you'll need to run the codegen again: `make proto-all` 

Changelog entry is probably a good idea I guess, no harm!

**Comment by crodriguezvega** [comment]
> I think you'll need to run the codegen again: make porto-all

Oh, yes, I always forget about this. Thanks for reminding! :)

**Comment by damiannolan** [comment]
Link checker is crying again 😢

**Review by colin-axner — APPROVED** [review]

**Pull request merged**